### PR TITLE
fix: automerge permissions with update version / disclaimer GH workflows

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR splits the approve and merge steps for the app version bump + update licences and disclaimer GH workflows. Previously with Mergify, the jobs used the default GH action bot to approve and tag with automerge. Since removing Mergify, we need to update the branch and automerge manually and ran into permissions issues with valora-bot trying to do too many things: https://github.com/valora-inc/wallet/actions/runs/3061163606

Now: valora-bot opens the PR, default GH bot approves, valora-bot merges.

### Other changes

Use conventional commit format for the opened PRs

### Tested

Example https://github.com/valora-inc/wallet/pull/2882

### How others should test

N/A

### Related issues

Relates RET-394

### Backwards compatibility

Yes

### CODE REVIEW

File: .github/scripts/createAppVersionPr.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ exec(
   -d '${JSON.stringify({
     head: branchName,
     base: 'main',
-    title: `Bump app version to ${appVersion}`,
+    title: `chore: bump app version to ${appVersion}`,
```
Comment (jeanregisser) [inline .github/scripts/createAppVersionPr.ts:34]: ❤️

File: .github/scripts/createLicenseDisclaimerPr.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ exec('git config user.name "valora-bot"')
 exec('git commit -m "Update licenses and disclaimer"')
 exec(`git push --set-upstream origin ${branchName}`)
 
-echo('Open licenses and disclaimer PR')
+echo('chore: open licenses and disclaimer PR')
```
Comment (jeanregisser) [inline .github/scripts/createLicenseDisclaimerPr.ts:24]: I think this change was meant to be a few lines down 😄

**Review by jeanregisser — APPROVED** [review]
Awesome! 💯

File: .github/scripts/createLicenseDisclaimerPr.ts (hunk 2 above)
Comment (kathaypacific) [reply .github/scripts/createLicenseDisclaimerPr.ts]: lollllll oops, brain fart. updated!

**Comment by ValoraQA** [comment]
Hey @MuckT since NA is mentioned in how other should test section ,we are skipping this PR 

CC: @kathaypacific 
Thanks.!

**Pull request merged**